.github/workflows/simulation_test.yaml

```diff
@@ -0,0 +1,180 @@
+
+    - name: Install libsonata
+      if: steps.cache-venv.outputs.cache-hit != 'true'
+      run: |
+        . ./venv/bin/activate
+        CC=mpicc CXX=mpic++ pip install git+https://github.com/BlueBrain/libsonata
+
+    - name: Install libsonatareport
+      run: |
+        mkdir build && cd build
+        cmake -DCMAKE_INSTALL_PREFIX=$(pwd)/install -DCMAKE_BUILD_TYPE=Release -DSONATA_REPORT_ENABLE_SUBMODULES=ON -DSONATA_REPORT_ENABLE_MPI=ON ..
+        cmake --build . --parallel
+        cmake --build . --target install
+        echo "SONATAREPORT_DIR=$(pwd)/install" >> $GITHUB_ENV
+
+    - name: Cache NEURON
+      id: cache-neuron
+      uses: actions/cache@v3
+      env:
+        cache-name: cache-neuron
+      with:
+        path: nrn
+        key: ${{ matrix.os }}-neuron-${{ env.NEURON_BRANCH }}-${{ env.NEURON_COMMIT_ID }}-py${{ matrix.python-version }}
+
+    - name: Install NEURON
+      if: steps.cache-neuron.outputs.cache-hit != 'true'
+      run: |
+        sudo apt-get install flex libfl-dev bison ninja-build
+        if [[ ${{ env.NEURON_BRANCH }} == 'master' ]]; then
+          git clone --branch=${{ env.NEURON_BRANCH }} https://github.com/neuronsimulator/nrn.git
+          cd nrn
+          git checkout ${{ env.NEURON_COMMIT_ID }}
+          cd ..
+        else
+          git clone --branch=${{ env.NEURON_BRANCH }} https://github.com/neuronsimulator/nrn.git --depth=1
+        fi
+        . ./venv/bin/activate
+        python -m pip install --upgrade pip -r nrn/nrn_requirements.txt
+        cmake -B nrn/build -S nrn -G Ninja \
+          -DPYTHON_EXECUTABLE=$(which python) \
+          -DCMAKE_INSTALL_PREFIX=$(pwd)/nrn/build/install \
+          -DNRN_ENABLE_MPI=ON \
+          -DNRN_ENABLE_INTERVIEWS=OFF \
+          -DNRN_ENABLE_CORENEURON=ON \
+          -DCMAKE_C_COMPILER=gcc \
+          -DCMAKE_CXX_COMPILER=g++ \
+          -DCORENRN_ENABLE_REPORTING=ON \
+          -DCMAKE_PREFIX_PATH=${{ env.SONATAREPORT_DIR }}
+        cmake --build nrn/build --parallel
+        cmake --build nrn/build --target install
+
+    - name: Install h5py
+      if: steps.cache-venv.outputs.cache-hit != 'true'
+      run: |
+        . ./venv/bin/activate
+        CC=mpicc CXX=mpic++ pip install --no-binary=h5py h5py
+
+    - name: Install neurodamus
+      run: |
+        . ./venv/bin/activate
+        git clone --branch=${{ env.PY_NEURODAMUS_BRANCH }} https://github.com/BlueBrain/neurodamus.git
+        cd neurodamus
+        pip install .
+
+    - name: Build models
+      run: |
+        export PATH=$(pwd)/nrn/build/install/bin:$PATH
+        # copy mod files from the Zenodo link
+        wget --output-document="O1_mods.xz" --quiet "https://zenodo.org/record/8026353/files/O1_mods.xz?download=1"
+        tar -xf O1_mods.xz
+        cp -r mod neurodamus/tests/share/
+        cp neurodamus/neurodamus/data/mod/*.mod neurodamus/tests/share/mod/
+        cd neurodamus
+        ./docker/build_neurodamus.sh tests/share/mod
```

Comment:
Neurodamus built mods from the paper publication because they were the only public models files when the pipeline was created. Now I think it may be better that we check out the mod files from the `neurodamus-models` repo. What do you think?